### PR TITLE
Adding external pushing capability.

### PR DESCRIPTION
**Pull request by VasuAgrawal**
This gives the buggy the ability to connect to the external server, sending data one way.

### CODE REVIEW

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java [hunk 1]
```diff
@@ -28,7 +29,7 @@ public static void main(String[] args) {
 		RobobuggyConfigFile.loadConfigFile(); //TODO make sure that logic Notification is setup before this point
 
 		new RobobuggyLogicNotification("Initializing Robot", RobobuggyMessageLevel.NOTE);
-        robot = SimRobot.getInstance();
+        robot = TransistorDataCollection.getInstance();
```
Comment (ramensandwich) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java]: Is there a consensus on which robot master should be running by default?

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java [hunk 2]
```diff
@@ -37,7 +38,10 @@ public static void main(String[] args) {
 
 		new RobobuggyLogicNotification("Starting Robot", RobobuggyMessageLevel.NOTE);
 		robot.startNodes();
-
+		
+		new RobobuggyLogicNotification("Connecting to external server", RobobuggyMessageLevel.NOTE);
+		new ExternalPusher("104.196.122.40", 8888);
```
Comment (ramensandwich) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java]: Please don't leave IP addresses and port numbers publicly available like this

**Comment by VivaanBahl** [comment]
@VasuAgrawal is this PR still relevant?

**Comment by VivaanBahl** [comment]
I am closing this until further notice, because I believe it is no longer relevant.
Please reopen if I am mistaken

**Pull request closed without merging**